### ISSUE TITLE
core-api pagination meta data prefix regression

### ISSUE THREAD

**Issue by kristjank** [ArkEcosystem/core#3378]
**Expected Behavior**

API Pagination meta data  should return self, next, previous, first and last paths without `api` prefix like this:
```json
{
  "meta": {
    "totalCountIsEstimate": true,
    "count": 100,
    "pageCount": 37251,
    "totalCount": 3725010,
    "next": "/transactions?transform=true&page=2&limit=100",
    "previous": null,
    "self": "/transactions?transform=true&page=1&limit=100",
    "first": "/transactions?transform=true&page=1&limit=100",
    "last": "/transactions?transform=true&page=37251&limit=100"
  },
```

**Current Behavior**
In the current behaviour the pagination meta data has the `api` prefix included. Like this:
```json
{
  "meta": {
    "next": "/api/transactions?transform=true&page=2&limit=100",
    "previous": null,
    "self": "/api/transactions?transform=true&page=1&limit=100",
    "first": "/api/transactions?transform=true&page=1&limit=100",
    "last": "/api/transactions?transform=true&page=37251&limit=100"
  },
```

The `api` part should not be there, when generating `request.meta` info for navigating.

**Comment by faustbrian** [comment]
Bit of context why the prefix has to be omitted. People might use nginx to serve their API under something like https://ark.myservice.com which routes to http://ip:4003/api so the API prefix wouldn't work so using a client SDK wouldn't work either.

The prefix has to be omitted so that clients can be used. The base URL of a client should include the `/api` prefix it shall be used.

**Comment by air1one** [comment]
Fixed by #3381
